pkg/crc/machine/sync.go

```diff
@@ -12,15 +12,13 @@ import (
 
 const startCancelTimeout = 15 * time.Second
 
-type State int
+type State string
```

Comment:
With my old C programmer hat on, I prefer `type State int` + `String()` method rather than `type State string`. Should not matter much here though.

Reply:
can you explain more about the reasoning of this preference? The emphasize is otherwise to strongly on 'I prefer' instead of what is better from the maintenance perspective.